### PR TITLE
New command Remove-DbaAgentJob

### PR DESCRIPTION
**Pull request by sanderstad**

### CODE REVIEW

File: functions/Remove-DbaAgentJob.ps1 [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                }
+                catch {
+                    Stop-Function -Message ("Something went wrong creating the job. `n$($_.Exception.Message)") -Target $instance
+                    return
+                }
+
+                # Delete the history
+                if (-not $KeepHistory) {
+                    Write-Message -Message "Purging job history" -Level Verbose
+                    $Job.PurgeHistory()
```
Comment (FriedrichWeinmann) [inline functions/Remove-DbaAgentJob.ps1]: All IO might fail, so try/catch it

**Comment by potatoqualitee** [comment]
hey @sanderstad  - can you give me an update on your PRs? are they ready to be rereviewed?

**Comment by sanderstad** [comment]
They should be ready. Didn’t I let you know in Slack?

Would it be better to leave a comment here in the future?

**Comment by potatoqualitee** [comment]
paaaaaaaaarty! 🎊

**Pull request merged**